server/views/pages/prisonerList/partials/searchTable.njk

```diff
@@ -0,0 +1,78 @@
+{% if not workAndInterests.problemRetrievingData %}
+  {{ govukTable({
+    attributes: {
+      'data-module': 'moj-sortable-table'
+    },
+    head: [
+      {
+        text: "Prisoner",
+        attributes: {
+          "aria-sort": "ascending"
+        }
+      },
+      {
+        text: "Location",
+        attributes: {
+          "aria-sort": "none"
+        }
+      },
+      {
+        text: "Release date",
+        attributes: {
+          "aria-sort": "none"
+        }
+      },
+      {
+        text: "Entered prison on",
+        attributes: {
+          "aria-sort": "none"
+        }
+      },
+      {
+        text: "Status",
+        attributes: {
+          "aria-sort": "none"
+        }
+      }
+    ],
+    rows: [
+      [
+        {
+          html: "<a href='/plan/G9981UK/view/overview' rel='noopener noreferrer' class='govuk-link govuk-link--no-visited-state'>Avary, Austin</a><br><span class='govuk-hint govuk-!-margin-bottom-0'>G9981UK</span>"
```

Comment:
May need to use the ugly `{% set ... %}` approach we used for the `futureGoalTargetDates` here!